### PR TITLE
Added `collator-set-up-guide.md`

### PR DESCRIPTION
**Pull request by PoisonPhang**
## Issue
N/A?

## Description
Converted [this](https://docs.google.com/document/d/1Ub76PGcS0dXWa0G-q064l9EKar15WPweLQHlAQiozjA/edit#heading=h.k7ffy1sj9xks) guide to markdown and added it to the mdbook.

**Note** image formatting isn't quite right, but that should be a matter for another PR.

## Checklist

- ~I have updated the cargo docs to reflect changes made by this PR~
- [X] I have updated the `book/` to reflect changes made by this PR

### CODE REVIEW

**Review by cor — DISMISSED** [review]
LGTM

File: book/src/developer-guides/collator-set-up-guide.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+```sh
+sudo apt install -y git clang curl libssl-dev llvm libudev-dev
+```
+
+### Setup Rust binary and Toolchain
+
+```sh
+#!/bin/bash
+
+RUST_C="nightly-2021-11-07"
```
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:29]: we switched to `nightly-2022-04-18`.

File: book/src/developer-guides/collator-set-up-guide.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+rustup toolchain install $RUST_C && \
+rustup target add wasm32-unknown-unknown --toolchain $RUST_C && \
+rustup default $RUST_C && \
+rustup show
+```
+
+### Get project and build node
+
+```sh
+git clone --depth 1 --branch v2.1.3 https://github.com/ComposableFi/composable.git && \
```
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:43]: the last release has a tag [v2.1.6](https://github.com/ComposableFi/composable/releases/tag/v2.1.6)

File: book/src/developer-guides/collator-set-up-guide.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+cd composable && \
+export SKIP_WASM_BUILD=1 && \
+cargo build --release
+```
+
+### One-liner
+```
```
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:50]:
```suggestion
```sh
```

**Review by 0xBrainjar2 — APPROVED** [review]

File: book/src/developer-guides/collator-set-up-guide.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
+![picasso-bannar](./picasso-bannar.png)
+
+# COLLATOR SET UP GUIDE
+
+In this document we will cover how to set up a collaborator with Composable
```
Comment (0xBrainjar2) [inline book/src/developer-guides/collator-set-up-guide.md:5]: collator

**Review by andor0 — APPROVED** [review]

**Pull request merged**